### PR TITLE
fix(bindings-ts): latest RPC & stellar-sdk

### PR DESCRIPTION
**Pull request by chadoh**
### What

- update to `stellar-sdk@11.0.1`
- fetch expiration using new `getFootprint` logic
- improve `initialize.sh` to show network status
  - latest RPC returns better error messages
  - deal with these and parse `getHealth` response

### Why

current `bindings typescript` logic is broken with latest Futurenet &
`quickstart:soroban-dev`

### Known limitations

Please double-check my `entryRes.entries[0].liveUntilLedgerSeq` logic in
the `getStorageExpiration` function.

### CODE REVIEW

File: cmd/crates/soroban-spec-typescript/src/project_template/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "dependencies": {
         "@stellar/freighter-api": "1.7.1",
         "buffer": "6.0.3",
-        "stellar-sdk": "11.0.0-beta.6"
+        "stellar-sdk": "11.0.0"
```
Comment (Shaptic) [inline cmd/crates/soroban-spec-typescript/src/project_template/package.json:7]: 11.0.1 has a bugfix related to getEvents! should be a trivial upgrade

File: cmd/crates/soroban-spec-typescript/src/project_template/src/assembled-tx.ts [hunk 2]
```diff
@@ -297,16 +297,15 @@ export class AssembledTransaction<T> {
   }
 
   getStorageExpiration = async () => {
-    const key = new Contract(this.options.contractId).getFootprint()[1]
-
-    const expirationKey = xdr.LedgerKey.expiration(
-      new xdr.LedgerKeyExpiration({ keyHash: hash(key.toXDR()) }),
+    const entryRes = await this.server.getLedgerEntries(
+      new Contract(this.options.contractId).getFootprint()
```
Comment (Shaptic) [inline cmd/crates/soroban-spec-typescript/src/project_template/src/assembled-tx.ts:301]: keep in mind that this doesn't include the wasm code expiration

**Review by Shaptic — APPROVED** [review]
small fixup needed but otherwise :shipit:

File: cmd/crates/soroban-spec-typescript/src/project_template/src/assembled-tx.ts (hunk 2 above)
Comment (chadoh) [reply cmd/crates/soroban-spec-typescript/src/project_template/src/assembled-tx.ts:301]: We previously used the persistent ttl specifically. How would I get that now? How would I get instance ttl?

Comment (Shaptic) [reply cmd/crates/soroban-spec-typescript/src/project_template/src/assembled-tx.ts:301]: this will give you the instance, just not the wasm

**Pull request merged**